modules/memberships/class-jetpack-memberships.php

```diff
@@ -42,6 +42,14 @@ class Jetpack_Memberships {
 	 * @var array
 	 */
 	private static $tags_allowed_in_the_button = array( 'br' => array() );
+
+	/**
+	 * The minimum required plan for this Gutenberg block.
+	 *
+	 * @var Jetpack_Memberships
+	 */
+	protected static $required_plan;
```

Comment:
Curious, what is the intention with setting visibility here to "protected" (vs "private" perhaps)? I might be missing a usage (or potential one) somewhere I guess.

Reply:
My thinking was mostly that a block's minimum required plan could be information that might be interesting to other code (outside the class itself). I guess I wasn't convinced enough to go totally `public` but limited the information to derived classes, which isn't a very strong line argument :sweat_smile: 

Applying my beloved YAGNI pattern, it probably makes sense to make it `private`.

```suggestion
	private static $required_plan;
```

Reply:
Very nice - thanks for clarifying. I think I'd go with `private` here too. :-)